pyhanko_certvalidator/fetchers/common_utils.py

```diff
@@ -85,7 +85,11 @@ def unpack_cert_content(
                 f"include a content type, assuming response body is a single "
```

Comment:
Can you update the content of this log message? It no longer reflects what the code is doing after your change :)

Reply:
Done! Please see if you like the new log message :)